fdbserver/VersionedBTree.actor.cpp

```diff
@@ -5400,13 +5571,15 @@ class VersionedBTree {
 		}
 	};
 
+#warning update forensic fields
```

Comment:
Did you intend to finish this in this PR?

Reply:
Ah, yes.  Warnings are how I track my TODO's.